modules/tile-converter/src/i3s-converter/helpers/load-3d-tiles.ts

```diff
@@ -65,3 +65,32 @@ export const loadTile3DContent = async (
 
   return tileContent;
 };
+
+/**
+ * Fetch 3DTiles tile content
+ * @param sourceTileset - source tileset that contain the tile
+ * @param sourceTile - source tile JSON metadata
+ * @param globalLoadOptions - tileset-wide load options
```

Comment:
Nit: Don't call them global if they are not actually global?
```suggestion
 * @param tilesetLoadOptions - tileset-wide load options
```